analyze.py

```diff
@@ -70,7 +69,7 @@ def is_fubar(results):
             has_wrong_pubkey = True
             logging.debug(conn['pubkey'][0] + ' is a fubar pubkey size')
             fubar = True
-        if ec_kex and pubkey_bits < min_ec_size:
+        if ec_kex and pubkey_bits < 160:
```

Comment:
That makes sense, I've pushed another commit. Just noticed that the has_good_pfs check in is_fubar still uses 160, but I've left that alone on the grounds that the DH parameter check is also only checking 1024, so not quite sure whether this is intentional.